docs/roadmaps/roadmap-2022.md

```diff
@@ -20,15 +20,35 @@ Please let us know if you have urgent needs which are not presented in the plan.
 - Scalability to support 1K TSP and 1K EPA			
 ### 0.4.0 [released]
 - UI to support EPA.					
-### 0.5.0 [May]
+### 0.5.0 [released]
 - Resource and Replicas Recommendation
 - Load-aware Scheduler
-### 0.6.0 [June]
+### 0.6.0 [released]
 - Scalability to support 3k TSP and 3k EPA		
 - Algorithm and QoS Documentation
 - EHPA grafana dashboard
+- DSP Algorithm Optimization
 ### 0.7.0 [July]
-- Support apiservice router for multiple metric adapters
-- Prediction with Business Metrics
+- Support APIService router for multiple metric adapters
+- Prediction with business metrics
+- Recommendation Framework
```

Comment:
- Support APIService router for multiple metric adapters
is replaced by `Support remote adapter for external metric` and released in 0.6.0
- Prediction with business metrics
is released in 0.6.0